applications/ws-irc3sp/public/v1/local.js

```diff
@@ -0,0 +1,45 @@
+const LETTERS = "ABCDEFHIJKLMNOPQRSTUVWXYZ";
+const SENTENCE_ENDING = ".?!";
+
+const sentences = (data, feed, ctx) => {
```

Comment:
could be moved to a generic package for reuse and documentation

Reply:
and tested ;-)

Reply:
> could be moved to a generic package for reuse and documentation

Yes, but which one ?
This is so specific, that it can't fit into `@ezs/basics`.
I thought of `@ezs/teeft` for a while, but the real purpose of this is to reduce the size of the texts to be analyzed by irc3sp.
That is to say, not spltting sentences containing abbreviated species name, so that they can be retrieved by the IRC3sp algorithm...
Maybe we could start an `@ezs/nlp` package, but even there, this sentence cutter would  not really be in its place, because of its specificity.
:thinking:

Reply:
@ezs/basics contains TXTParser which allows to split text into segments, we can imagine adding TXTSentences which splits into sentences.
Otherwise you need to create a new package which will contain tools for text

Reply:
It was eventually integrated into [`@ezs/strings`](https://inist-cnrs.github.io/ezs/#/plugin-strings?id=sentences).